### PR TITLE
Add Bi Shape Triangle

### LINKED ISSUE #744: Less shapes available for sketch
There are currently 8 shapes for sketching. There should be more sketches in that section like bidirectional connector, ellipse, rectangle, etc.
Capture

I am a GSSoC'21 participant and I would like to work on this issue. May I work on it under GSSoC'21?

### PR DESCRIPTION
**Pull request by alkatrivedi**
### Changes
- Added a Bi Shape Triangle(Right Angle Triangle) in sketch page shapes.

Capture

Closes #744

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Do implement all the features of the edge styles present there

**Comment by smaranjitghose** [comment]
Wait for [this](https://github.com/smaranjitghose/doc2pen/pull/735) to be merged and then incorporate the changes

**Comment by alkatrivedi** [comment]
> Do implement all the features of the edge styles present there

I tried to find out the Bi Shape Triangle with the same edge style as present there. But this was the only style for Bi Shape Triangle I found in react icons. I have searched a lot but couldn't find one with the same edge style. If there's any source to get Bi Shape Triangle with the same edge style could you please tell me?

File: src/pages/Sketch/components/Canvas.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function Canvas() {
 
     if (type === "pen") {
       logicDown(point);
-    } else if (["line", "square", "circle", "triangle", "arrow", "diamond"].includes(type)) {
+    } else if (["line", "square", "circle", "triangle", "arrow", "diamond", "righttriangle"].includes(type)) {
```
Comment (iamdarshshah) [inline src/pages/Sketch/components/Canvas.js:118]:
```suggestion
    } else if (["line", "square", "circle", "triangle", "arrow", "diamond", "rightTriangle"].includes(type)) {
```
Use camelCase.

File: src/pages/Sketch/components/Canvas.js [hunk 2]
```diff
@@ -363,6 +366,34 @@ function Canvas() {
     context.stroke();
   }
 
+
+  function righttriangleMove(point) {
```
Comment (iamdarshshah) [inline src/pages/Sketch/components/Canvas.js:370]:
```suggestion
  function rightTriangleMove(point) {
```
Update this too.

File: src/pages/Sketch/components/Canvas.js (hunk 1 above)
Comment (alkatrivedi) [reply src/pages/Sketch/components/Canvas.js:118]: Sure.

File: src/pages/Sketch/components/Canvas.js (hunk 2 above)
Comment (alkatrivedi) [reply src/pages/Sketch/components/Canvas.js:370]: Yeah.

File: src/pages/Sketch/components/Canvas.js (hunk 1 above)
Comment (alkatrivedi) [reply src/pages/Sketch/components/Canvas.js]: Done with the changes.

File: src/pages/Sketch/components/Canvas.js (hunk 2 above)
Comment (alkatrivedi) [reply src/pages/Sketch/components/Canvas.js]: Done.

File: src/pages/Sketch/components/Toolbox/Toolbox.js [hunk 3]
```diff
@@ -68,6 +69,9 @@ function Toolbox({
                 <Shape type_="text" id="sketch-shapes-text" label="Text">
                     <FaFont size={15}/>
                 </Shape>
+                <Shape type_="rightTriangle" id="sketch-shapes-righttriangle" label="Bi Shape Triangle">
```
Comment (iamdarshshah) [inline src/pages/Sketch/components/Toolbox/Toolbox.js:72]:
```suggestion
                <Shape type_="rightTriangle" id="sketch-shapes-rightTriangle" label="Bi Shape Triangle">
```
I know this is not at all necessary but it's better to update it everywhere it is been used. Just to make things look simpler.

Comment (alkatrivedi) [reply src/pages/Sketch/components/Toolbox/Toolbox.js:72]: Yeah sure.

Comment (alkatrivedi) [reply src/pages/Sketch/components/Toolbox/Toolbox.js]: Done.

**Review by iamdarshshah — APPROVED** [review]
@anushbhatia Are there any other changes that you might want to suggest @alkatrivedi to include in this PR? 

Otherwise, it LGTM! Nice work @alkatrivedi 🎉 

> Note: As mentioned by @smaranjitghose - we don't have to merge it right away. Please wait until #735 gets merged.

**Comment by alkatrivedi** [comment]
Thank you @iamdarshshah 😃

**Comment by smaranjitghose** [comment]
@alkatrivedi Could you please incorporate the recent changes and update the PR with a squash commit. The rest looks cool!

**Comment by alkatrivedi** [comment]
> @alkatrivedi Could you please incorporate the recent changes and update the PR with a squash commit. The rest looks cool!

Yeah sure.

**Comment by alkatrivedi** [comment]
My HEAD is lost and I'm not able to redirect it to the correct point. Please help me with what should I do?
Capture

**Comment by alkatrivedi** [comment]
> @alkatrivedi Could you please incorporate the recent changes and update the PR with a squash commit. The rest looks cool!

I have done the changes please have a look @smaranjitghose. Thank you.

**Comment by smaranjitghose** [comment]
@alkatrivedi

**Comment by alkatrivedi** [comment]
Thank you @smaranjitghose.

**Pull request merged**